aeacus-reports.pl

```diff
@@ -273,8 +274,9 @@ =head1 DESCRIPTION
     }
 }
 
+# Do not upload to SweStore on Irma
 my $swestoreFlag = "-swestore";
-if ($sisyphus->machineType() eq 'hiseqx'){
+if (hostname() eq 'irma'){
```

Comment:
I presume `hostname()` will return `irmaX`, where `X` is the unique host identifier, so this will then not work. You probably want to do something like `if (index(hostname(), "irma") != -1)`.